.github/workflows/docker.yml

```diff
@@ -95,21 +95,15 @@ jobs:
     strategy:
       matrix:
         include:
-          - platform: linux/amd64
-            digest: amd64-slim
-            bim_support: false
-            debian_base: trixie
-            target: slim
-            runner: runner=4cpu-linux-x64
           - platform: linux/amd64
             digest: amd64-bim
             bim_support: true
             debian_base: bookworm
-            target: slim-bim
+            target: slim
```

Comment:
The CI closed the last PR on which I added a remark for this, but this merge includes a commit that was intended to only target 16.6: https://github.com/opf/openproject/pull/20777/commits/533ee53d47849b11619e415654657437f500a3e9

CC @oliverguenther

Reply:
I reverted that revert now, so that the change stays out of 16.6, but is in for 17.0.